src/vite-env.d.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="vite/client" />
+
+declare const FS_URL: string
```

Comment:
Why is this called `FS`?

Where will the `FS_URL` point to on the website? I am not able to find that out...

Reply:
The name may be a bit misleading. In dev mode, `FS_URL` points to a simple server that serves the `cache` directory. The server is setup as a Vite plugin in https://github.com/enso-org/engine-benchmark-results/blob/4e9f6bf8928a721a590347bb14f07b19694d84e7/vite.config.ts#L17 in dev mode.

In production, this server is not used, but the contents of the `cache` directory is copied inside the `dist` dir that is supposed to be served by the GH pages. Copy is implemented via `vite-plugin-static-copy` in https://github.com/enso-org/engine-benchmark-results/blob/4e9f6bf8928a721a590347bb14f07b19694d84e7/vite.config.ts#L21-L24